src/main/java/duke/commands/AddCommand.java

```diff
@@ -0,0 +1,72 @@
+package duke.commands;
```

Comment:
I think it is preferred to use singular form for package naming, i.e. `package duke.command;`.